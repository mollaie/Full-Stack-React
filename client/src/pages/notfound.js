const NotFound = () => {
  return <div className="not-found-container">Not Found is working</div>;
};

export default NotFound;
