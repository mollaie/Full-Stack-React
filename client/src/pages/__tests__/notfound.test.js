import React from "react";
import renderer from "react-test-renderer";
import notfound from "../notfound";

describe("notfound test", () => {
  it("notfound should match snapshot", () => {
    const component = renderer.create(<notfound />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
