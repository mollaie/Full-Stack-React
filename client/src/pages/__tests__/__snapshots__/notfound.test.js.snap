// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`notfound test notfound should match snapshot 1`] = `<notfound />`;
